keras_cv/models/stable_diffusion/stable_diffusion.py

```diff
@@ -87,14 +132,15 @@ def __init__(
         self,
         img_height=512,
         img_width=512,
+        version='v1',
```

Comment:
actually, because this is a new architecture we might want to make this a different model:

```
class StableDiffusionv2
```

That's what we do for ResNet, MobileNet, etc